cmd/podman/common/build.go

```diff
@@ -65,11 +65,11 @@ func DefineBuildFlags(cmd *cobra.Command, buildOpts *BuildFlagsWrapper) {
 
 	// --pull flag
 	flag := budFlags.Lookup("pull")
-	if err := flag.Value.Set("true"); err != nil {
-		logrus.Errorf("Unable to set --pull to true: %v", err)
+	flag.DefValue = "missing"
+	if err := flag.Value.Set("missing"); err != nil {
```

Comment:
Unless I'm mistaken, this is different than Docker which does a `pull=true" by default.